ovve_ui.py

```diff
@@ -90,38 +94,42 @@ def updateValue(self, value):
 
 
 class DisplayRect(QWidget):
-    def __init__(self, label, value, unit, parent=None, size=(200, 100)):
+    def __init__(self,
+                 label,
+                 value,
+                 unit,
+                 rect_settings,
+                 parent=None,
+                 size=(200, 100)):
```

Comment:
Can rect_settings contain the size?

Reply:
My intention was that the rect_settings were unified across the entire UI to allow for styling all boxes at once. That was why I omitted size.